api/envoy/config/trace/v3/datadog.proto

```diff
@@ -16,6 +18,19 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: Datadog tracer]
 
+// Configuration for the Remote Configuration feature.
+message DatadogRemoteConfig {
+  // Allows to configure and change the tracer behaviour from Datadog's user interface.
+  // Default value is ``false``, which disable the feature.
+  // [#comment: This feature can drastically increase the number of connections to the Datadog Agent.
+  // Each tracer regularly polls for configuration updates, and the number of tracers is the product
+  // of the number of listeners and worker threads.]
+  bool enabled = 1;
+
+  // Frequency at which new configuration updates are queried.
+  google.protobuf.Duration polling_interval = 2;
```

Comment:
Is there a default for this if not specified? It doesn't look like there is in the code. I would either make this required and add a PGV annotation or supply a default and document it.

Reply:
If this is the case you can leave it like it is, but you should document that behavior